spec/unit/trello_wrapper_spec.rb

```diff
@@ -47,6 +47,36 @@
     end
   end
 
+  describe '#backup' do
+    it 'raises an error for empty board id' do
+      expect { subject.backup("") }.to raise_error(TrolloloError)
+    end
+
+    it 'raises an error for nil board id' do
+      expect { subject.backup(nil) }.to raise_error(TrolloloError)
+    end
```

Comment:
For consistency there also should be a test for successful execution of the `backup` method as it's done for the `retrieve_board_data` method. Or there should be none.

It's actually debatable if testing that the correct HTTP request is done is useful on this level, because the test mostly just duplicates the implementation and it's unlikely to break by a change unrelated to changing the request itself (which would require a change of the test as well). Maybe it would be better to rely on tests on a higher level which use these methods and then actually process the data returned by these calls.